app/public/js/home.js

```diff
@@ -55,8 +55,12 @@ function initMap()
 		streetViewControl: false,
 		clickableIcons: false
 	});
-	default_location = new google.maps.LatLng(default_location.latitude, default_location.longitude);
-
+	try{
+		default_location = new google.maps.LatLng(default_location.latitude, default_location.longitude);
```

Comment:
When was this needed?  What error did you run into?  Was default_location.latitude undefined at some point?